src/frontend/src/lib/services/exchange.services.ts

```diff
@@ -36,10 +37,21 @@ export const exchangeRateERC20ToUsd = async (
 		include_market_cap: true
 	});
 
+export const exchangeRateICRCToUsd = async (
+	contractAddresses: LedgerCanisterIdText[]
+): Promise<CoingeckoSimpleTokenPriceResponse | null> =>
+	simpleTokenPrice({
+		id: 'internet-computer',
+		vs_currencies: 'usd',
+		contract_addresses: contractAddresses.map((address) => address.toLowerCase()),
```

Comment:
ICRC token have addresses?

Reply:
in this case I am keeping the coingecko API convention. They define this function with an input called contract address. But in the rest of the code, I changed to canisterId

Reply:
yes, makes sense, changed